.github/workflows/base-commit-message-checker.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: 'Commit Message Check'
+
+on:
+  workflow_call:
+    secrets:
+      accessToken:
+        required: true
+
+jobs:
+  base-check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Subject Begining
```

Comment:
There are still some typo fixes and lowercase/uppercase differences that weren't updated here

Reply:
Brought the changes from master here. Resolving.